code/datums/mob_stats.dm

```diff
@@ -238,6 +238,15 @@
 	name = STAT_VIG
 	desc = "Here, paranoia is nothing but a useful trait. Improves your ability to control recoil on guns, helps you resist insanity."
 
+/datum/stat/vivification
+	name = STAT_VIV
+	desc = "The body below the flesh can take only so much stimulation, takes a lot to train the body to handle drugs all be it helpful or harmful."
```

Comment:
what exactly is the "body below the flesh". bodies be made out of flesh
, takes -> , it takes
drugs all be it -> drugs, be they